apps/user/schema.py

```diff
@@ -0,0 +1,60 @@
+from typing import Union
+
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django_extras import DjangoObjectField, PageGraphqlPagination
+
+from utils.graphene.types import CustomDjangoListObjectType
+from utils.graphene.fields import DjangoPaginatedListObjectField
+
+from .models import User
+from .filters import UserFilterSet
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = User
+        fields = (
+            'id', 'email', 'first_name', 'last_name', 'is_active'
+        )
+
+    display_name = graphene.String()
+
+    @staticmethod
+    def resolve_display_name(root, info, **kwargs):
+        return root.profile.get_display_name()
+
+
+class MeUserType(UserType):
+    class Meta:
+        model = User
+        fields = (
+            'id', 'email', 'first_name', 'last_name',
+            'is_active', 'last_login',
+        )
+
+
+# XXX: Is this okay to do?
+class UserListType(CustomDjangoListObjectType):
+    base_type = UserType
```

Comment:
Is this possible?

Reply:
Yup. @annshress Is this okay to do?

https://github.com/the-deep/server/pull/836/files#diff-ac5f6b70238d8921b24e3faace6afd8a43c50a480f3f5257d94d45c4c2e0444aR126

Reply:
This is removed.